ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template7.kt

```diff
@@ -368,7 +366,7 @@ private fun Features(
     val localization = selectedLocalizationForTier(selectedTier)
 
     AnimatedContent(
-        targetState = selectedTier,
+        targetState = localization,
```

Comment:
Changed this, since a linter was complaining if we weren't using the state within the `AnimatedContent` block.

Reply:
That's good! Leads to subtle bugs if you don't use the state.